redis/example/kustomization.yaml

```diff
@@ -24,10 +40,11 @@ patches:
         path: /spec/template/spec/containers/0/resources/limits/memory
         value: "2Gi"
 
+
 # Redis password secret
 secretGenerator:
   - name: redis
     envs:
-      - secrets/redis-secret.txt
+      - secrets/redis-secret.yaml
```

Comment:
nit/unrelated to the architecture change: but why has this changed? Were the contents of this file always in YAML?

Reply:
[oh, right, that was silly change](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/68d08076192f871ea58d454ee525adc6899e4284#diff-29fa0c5dc8aa4d8d05f166a67bcef458ae148967740033b27d20f1313a3e43dcR48)